src/components/common/CreatableMultiSelect.tsx

```diff
@@ -20,7 +20,7 @@ const S = styled(CreatableSelect)<styledProps>`
   user-select: none;
   font-size: 12px;
   border-width: 0px !important;
-
+  z-index: 900;
```

Comment:
this is a brittle solution what happens if we have another modal or screen that needs to be on top?